src/armado/compressed_index.py

```diff
@@ -365,7 +357,9 @@ def __init__(self, directory):
         #   docsets = FrozenStringList
         keyfilename = os.path.join(directory, "compindex.key.bz2")
         fh = CompressedFile(keyfilename, "rb")
-        matrix, docsets = cPickle.load(fh)
+
+        # TODO: research and document why the encoding here is latin-1
+        matrix, docsets = pickle.load(fh, encoding='latin-1')
```

Comment:
¿Se necesita éste encoding?
```suggestion
        matrix, docsets = pickle.load(fh)
```

Reply:
Si es necesario o al menos eso entendí cuando lo cambié, pero no me acuerdo por qué. Puse un comentario en la linea de arriba para que alguien se ponga a investigarlo. Está relacionado a como pickle guarda los arrays. Si te parece intentar ver por qué genial!